tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/VersionConflict/VersionConflict2xTests.cs

```diff
@@ -88,9 +87,10 @@ public void SubmitTraces()
                 var runtimeId = rootSpan.Tags[Tags.RuntimeId];
                 Guid.TryParse(runtimeId, out _).Should().BeTrue();
 
-                httpSpans.Should().OnlyContain(
-                    s => s.Tags[Tags.RuntimeId] == runtimeId,
-                    "runtime id should be synchronized across versions of the tracer");
+                httpSpans.Should()
+                         .OnlyContain(
+                              s => s.Tags[Tags.RuntimeId] == runtimeId,
+                              "runtime id should be synchronized across versions of the tracer");
```

Comment:
This whole file is just reformatting?

Reply:
yes :( my bad, fixed in b9b2c8c95e0dc5c966184933b404f5d1ed7d06ba